leptos/Cargo.toml

```diff
@@ -21,42 +21,40 @@ typed-builder-macro = "0.16"
 serde = { version = "1", optional = true }
 serde_json = { version = "1", optional = true }
 server_fn = { workspace = true }
-web-sys = { version = "0.3.63", optional = true }
+web-sys = { version = "0.3.63", features = ["ShadowRoot", "ShadowRootInit", "ShadowRootMode"] }
```

Comment:
This change also pulled in a bunch of formatting changes to this `Cargo.toml` when you saved it, presumably due to an editor plugin with different formatting settings or something.